src/components/sponsers/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { charityAPI } from '../../clients';
+import './styles.css';
+
+class Sponsers extends React.Component {
+  state = { urls: [], urlsError: false };
+  componentDidMount() {
+    charityAPI('/Sponsers')
+      .then(res => {
+        console.log(res);
+        this.setState({
+          urls: res.data
+        });
```

Comment:
After we received the response you changed the state of `urls`.
You also need to change the state of `error `and `loading`.
_whether you received the **data you expect** or **error** it means that no more loading_ so you need to set the state of` loading: false` and `error :false`